src/main/webapp/static/js/web3provider.js

```diff
@@ -39,14 +39,7 @@ function init() {
         // Mikko's test key - don't copy as your mileage may vary
         infuraId: "8043bb2cf99347b1bfadfb233c5325c0",
       }
-    },
-
-    fortmatic: {
-      package: Fortmatic,
-      options: {
-        // Mikko's TESTNET api key
-        key: "pk_test_391E26A3B43A3350"
-      }
+    }
     }
```

Comment:
Also delete this line, to avoid `Uncaught SyntaxError: Unexpected token '}'`

Reply:
Thx, for the fix!